packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -274,7 +274,9 @@ describe('<DataGrid />', () => {
             <DataGrid {...defaultProps} />
           </div>,
         );
-        const header = screen.getByRole('columnheader', { name: 'brand' });
+        const header = screen
+          .getByRole('columnheader', { name: 'brand' })
+          .querySelector('.MuiDataGrid-colCellTitleContainer');
```

Comment:
Does it mean we have broken the a11y feature?

Reply:
No the sorting is just on the title as we now have different buttons in the cell